### ISSUE TITLE
ACE Interaction Menu not working

### ISSUE THREAD

**Issue by oonorlund** [acemod/ACE3#3645]
**Arma 3 Version:** `1.56.134787`
**CBA Version:** `2.3.1`
**ACE3 Version:** `3.5.1`

**Mods:**
- `@CBA_A3`
- `@ace`

**Description:**
- Both in the 3D editor and playing multiplayer on my group's server whenever I try to use the ACE interaction, the menu shows up for a split second and then disappears.  As such, I am unable to use the ACE menu to interact either with myself or others.

**Steps to reproduce:**
-Just logging into the 3D editor or the group's server and trying to use ACE.

**Where did the issue occur?**
- Both in single-player with the 3D editor or on our group's multiplayer server

**Placed Modules:**
- Error occurs even with no modules down in the 3D editor

**RPT log file:**
- http://pastebin.com/N9Q7CDkJ

**Comment by jonpas** [comment]
Can you please upload the RPT to Gist or Pastebin (as was described in the issue template)? It will make this issue much more readable. Thanks!

First of all, please try without ACE3 optionals, the 2 you are running require RHS:
> ace_compat_rhs_afrf3.pbo
> ace_compat_rhs_usf3.pbo

**Comment by oonorlund** [comment]
http://pastebin.com/N9Q7CDkJ

**Comment by commy2** [comment]
rebind to a different key

**Comment by Gaspasar** [comment]
I had the same problem at the beginning with ace.
It was a problem with my keyboard and win 10.
The windows key is not able to "hold" down at my k95.
I assigned it to the "space" key. (And ctrl+space)

**Comment by oonorlund** [comment]
I did some more work on this with the guy that runs our server, and we found that it was an issue with the Arma profile.  Removing the RHS compat files did nothing.  Changing the keybinds did nothing.  It was an issue with the Arma profile I was using.  What that issue is, we don't know.  But feel free to close this thread if you want.

If you are interested, these are the report logs with the RHS compat files removed:
http://pastebin.com/DDjbxf0L
